pkg/registry/registry_aliyun.go

```diff
@@ -64,7 +65,7 @@ func (r *Registry) GetAliyunTagsWithDetails(image string) (tags []QuayTag, err e
 	}
 
 	tagRequest := cr.CreateGetRepoTagsRequest()
-	tagRequest.Domain = "cr.cn-shanghai.aliyuncs.com"
+	tagRequest.Domain = fmt.Sprintf("cr.%s.aliyuncs.com", r.aliyunRegion)
```

Comment:
Let's turn this `fmt.Sprintf` into a little function in `key.go` in a follow up PR.